.devcontainer/devcontainer.json

```diff
@@ -2,7 +2,7 @@
   "name": "caltrans/pems",
   "dockerComposeFile": ["../compose.yml"],
   "service": "dev",
-  "runServices": ["dev"],
+  "runServices": ["dev", "docs"],
   "workspaceFolder": "/home/caltrans/src",
```

Comment:
If you add:

```json
"forwardPorts": ["docs:8000"],
```

Then the link to the `docs` service will show up in the devcontainer in the `Ports` tab:

![image](https://github.com/user-attachments/assets/27bba86a-f5a9-495f-ba25-f017fd494043)

This is helpful to quickly get to the docs site locally.